semantic-python/test/Instances.hs

```diff
@@ -25,7 +25,8 @@ instance ToJSON a => ToJSON (File a) where
     , "body" .= fileBody
     ]
 
-deriving newtype instance ToJSON Path
+instance ToJSON Path.AbsRelFile where
+  toJSON p = toJSON (pack (show p))
```

Comment:
This isn’t correct; you want `Path.toString` instead of `show`.

```
λ import qualified System.Path  as Path
λ let p = Path.relFile("./cabal.project")
λ show p
"currentDir </> relPath \".\" </> relPath \"cabal.project\""
λ Path.toString p
"./cabal.project”
```